deploy/helms/legion/values.yaml

```diff
@@ -158,4 +158,6 @@ connections:
 
 secrets:
   nexus:
-    admin: "jonny"
\ No newline at end of file
+    admin: "jonny"
+  token_secret: "bGVnaW9uLXNlY3JldAo=" # "legion-secret" is default value
+  token_ttl_minutes: "MTIw" # 120 minutes is default TTL
```

Comment:
Why do you use base64 encoded value? this looks so strange

Reply:
Do we need a default value for a secret in the first place? This is a such opportunity for a security hole.